backend/src/plugins/ModActions/functions/actualKickMemberCmd.ts

```diff
@@ -77,6 +77,17 @@ export async function actualKickMemberCmd(
     },
   });
 
+  if (pluginData.config.getForMember(msg.member).delete_invites_on_kick) {
+    const userInvites = (await pluginData.guild.getInvites()).filter(inv => inv.inviter?.id === user.id);
+    userInvites.forEach(invite => {
```

Comment:
After adding `await` below for `invite.delete()`, it might be best to do a `for (const invite of userInvites)` loop instead and bail from the loop at the first deletion error. The current code would attempt to delete the invites in parallel due to `forEach` which could potentially cause a lot of errors.